README.md

```diff
@@ -1 +1,54 @@
-# Hello-world
\ No newline at end of file
+# Hello-world
+
+ this project, is setted up for Learning Purpose .No complex coding is required for this exercise. the goal here is to master all of this tools (git,github workflow and linter).
+
+ ### Built With
+
+ - Html,css
+ - No framework used
+ - Lint
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+Clone or Download Repository
+Extract the download Folder
+
+### Install
+
+No need of Installation
+
+### Usage
+
+Open the extracted folder and double click on index.html. BOOM!!
+
+and for the contributer i recommended to use [VSCode](https://code.visualstudio.com/) as text editor and Live Server installed on it.
+
+👤 **Author**
+
+- GitHub: [@freddykaberuka](https://github.com/freddykaberuka)
+- Twitter: [@trojanx07](https://twitter.com/trojanx07)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kaberuka-freddy-853b08153/)
```

Comment:
- [ ] Kindly write your name instead of "author" and give this section a title like in the [README template](https://github.com/microverseinc/readme-template) provided by Microverse.